<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDataFrom</class>
 <widget class="QWidget" name="QDataFrom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pb_data_pgup">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_data_pgdn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_data_printer">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_data_delete">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_data_export">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>170</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_data_cloud">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>220</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_clean_page">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_clean_jump">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>320</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
  </widget>
  <widget class="QTableView" name="tb_data">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>140</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
